requirements/base.in

```diff
@@ -2,7 +2,8 @@
 
 -c constraints.txt
 
-Django>=1.11,<2.0          # Web application framework
+Django>=1.11                        # Web application framework
+Django<2.0; python_version < "3"    # Web application framework
```

Comment:
We'll need to remove the upper limit on the version, the environment marker gets applied incorrectly in the generated files due to a pip-tools bug.